src/Streamly/Data/Unfold.hs

```diff
@@ -94,7 +94,6 @@ module Streamly.Data.Unfold
     , dropWhileM
 
     -- ** Zipping
-    , zipWithM
```

Comment:
Any reason for not exporting this?

Reply:
Just exporting a minimal essential ones in this release, we can export others later. Anyway they are available if someone needs them.